client/src/models/EventSeries.ts

```diff
@@ -0,0 +1,124 @@
+import { gql } from "@apollo/client"
+import Model, { GRAPHQL_ENTITY_AVATAR_FIELDS } from "components/Model"
+import _isEmpty from "lodash/isEmpty"
+import EVENT_SERIES_ICON from "resources/eventSeries.png"
+import Settings from "settings"
+import utils from "utils"
+import * as yup from "yup"
+
+export default class EventSeries extends Model {
+  static resourceName = "EventSeries"
+  static listName = "eventSeriesList"
+  static getInstanceName = "eventSeries"
+  static relatedObjectType = "eventSeries"
+
+  static displayName() {
+    return "Event Series"
+  }
+
+  static schema = {}
+
+  static yupSchema = yup.object().shape({
+    status: yup
+      .string()
+      .required()
+      .default(() => Model.STATUS.ACTIVE),
+    name: yup.string().required().default(""),
+    description: yup.string().default(""),
+    hostOrg: yup
+      .object()
+      .test("hostOrg", "host org error", (hostOrg, testContext) =>
+        _isEmpty(hostOrg)
+          ? testContext.createError({
+            message: `You must provide the ${Settings.fields.eventSeries.hostOrg.label}`
+          })
+          : true
+      )
+      .default({}),
+    adminOrg: yup
+      .object()
+      .test("adminOrg", "admin org error", (adminOrg, testContext) =>
```

Comment:
Can the event-series administration organization be made optional?

Reply:
Is optional now.